storage/storage.hpp

```diff
@@ -505,8 +505,14 @@ class Storage
                                                   TCountriesVec const & descendants,
                                                   MapFilesDownloader::TProgress const & downloadingMwmProgress,
                                                   TCountriesSet const & mwmsInQueue) const;
+
+  template <class ToDo>
+  void ForEachInSubtreeAndInQueue(TCountryId const & root, ToDo && toDo) const;
 };
 
+void GetQueuedCountries(Storage::TQueue const & queue, TCountriesSet & resultCountries);
+bool HasCountryId(TCountriesVec const & sorted, TCountryId const & countyId);
```

Comment:
Parameter names here differ from the source. In any case, please, rename sorted by sortedIds.